core/tests/webdriverio_utils/PreferencesPage.js

```diff
@@ -82,44 +92,54 @@ var PreferencesPage = function() {
 
   this.editUserBio = async function(bio) {
     await action.addValue('User bio field', userBioElement, bio);
+    // Click on a neutral element to make User bio element out of focus.
+    await action.click('Preferences page header', pageHeader);
+    await clickSaveChangesButton();
     await saveNewChanges('User Bio');
   };
 
   this.toggleEmailUpdatesCheckbox = async function() {
     await action.click('Email Updates checkbox', emailUpdatesCheckbox);
+    await clickSaveChangesButton();
     await saveNewChanges('Email Updates');
   };
 
   this.toggleEditorRoleEmailsCheckbox = async function() {
     await action.click('Editor role emails checkbox', editorRoleEmailsCheckbox);
+    await clickSaveChangesButton();
     await saveNewChanges('Editor Role Emails');
   };
 
   this.toggleFeedbackEmailsCheckbox = async function() {
     await action.click(
       'Feedback emails checkbox', feedbackMessageEmailsCheckbox);
+    await clickSaveChangesButton();
     await saveNewChanges('Feedback Emails');
   };
 
   this.selectSystemLanguage = async function(language) {
     await action.click('system language selector', languageSelector);
     var dropdownOption = $(`.mat-option-text=${language}`);
     await action.click('clickable', dropdownOption);
+    await clickSaveChangesButton();
     await saveNewChanges('System Language');
   };
 
   this.selectPreferredAudioLanguage = async function(language) {
     await action.click('clickable', audioLanguageSelector);
     var dropdownOption = $(`.mat-option-text=${language}`);
     await action.click('clickable', dropdownOption);
+    await clickSaveChangesButton();
     await saveNewChanges('Preferred Audio Language');
   };
 
   this.setUserBio = async function(bio) {
     var inputFieldName = 'User bio input field';
     await action.clear(inputFieldName, userBioElement);
-    await saveNewChanges('User Bio');
     await action.setValue(inputFieldName, userBioElement, bio);
+    // Click on a neutral element to make User bio element out of focus.
```

Comment:
Why is this needed? Also how is setUserBio different from editUserBio?

Reply:
'setUserBio' replaces the old value with the given value.
'editUserBio' just appends the given string to the older bio.
I have preserved the functionality here.

Reply:
OK. I think it's fine to keep as-is, please just add JSDoc if that's OK so that future devs are not confused.

Reply:
done

Reply:
Not done. Please don't say something is done when it isn't.

Reply:
Sorry for slight confusion --- I thought you have asked JSDoc for this lines 
'// Click on a neutral element to make User bio element out of focus'
'await action.click('Preferences page header', pageHeader);' as you have commented on this lines. So I have put 'Done' as I have removed those 2 lines to 'saveNewChanges' function.

Got it now. Will make a comment explaining what is 'setUserBio' and 'editUserBio'